test/unittest/blas3/blas3_gemm_test.cpp

```diff
@@ -116,15 +116,14 @@ void run_test(const combination_t<scalar_t> combi) {
           m_b_gpu + _base(dim_b, ldb_mul, bs), ldb, beta,
           m_c_gpu + _base(dim_c, ldc_mul, bs), ldc);
 
-    auto event = policy_handler.copy_to_host(
-        m_c_gpu + _base(dim_c, ldc_mul, bs),
-        c_m_gpu.data() + _base(dim_c, ldc_mul, bs), _size_batch(dim_c, ldc_mul));
+    auto event =
+        policy_handler.copy_to_host(m_c_gpu + _base(dim_c, ldc_mul, bs),
+                                    c_m_gpu.data() + _base(dim_c, ldc_mul, bs),
+                                    _size_batch(dim_c, ldc_mul));
     policy_handler.wait(event);
   }
 
-  for (int i = 0; i < _size(dim_c, ldc_mul); ++i) {
-    ASSERT_T_EQUAL(scalar_t, c_m_gpu[i], c_m_cpu[i]);
-  }
+  ASSERT_TRUE(utils::compare_vectors(c_m_gpu, c_m_cpu));
```

Comment:
You can call this ASSERT_NEAR or ASSERT_ALMOST_EQUAL

Reply:
What do you mean? Do you want me to get the threshold and use ASSERT_NEAR on it?

Reply:
We discussed it offline, you can ignore that